### PR TITLE
Update SHA checksum for operator image 5.0.10 to match quay.io

### PR DESCRIPTION
**Pull request by pjuarezd**
* Infer the Operator version from the constants.
* Update Docker Images SHA from repo

### CODE REVIEW

**Comment by pjuarezd** [comment]
PR's to publish in redhat marketplaces
https://github.com/redhat-openshift-ecosystem/certified-operators/pull/2973
https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators/pull/574

**Comment by harshavardhana** [comment]
Uggh

```
2s
Run yarn audit --groups dependencies
yarn audit v[1](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:1).22.19
warning Lockfile has incorrect entry for "debug@^[4](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:5).1.0". Ignoring it.
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ critical      │ Babel vulnerable to arbitrary code execution when compiling  │
│               │ specifically crafted malicious code                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ @babel/traverse                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=7.23.2                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ mds                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ mds > styled-components > @babel/traverse                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1094349                     │
└───────────────┴──────────────────────────────────────────────────────────────┘
1 vulnerabilities found - Packages audited: 306
Severity: 1 Critical
Done in 1.[5](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:6)7s.
Error: Process completed with exit code 1[6](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:7).
```

**Review by harshavardhana — APPROVED** [review]

**Comment by pjuarezd** [comment]
> Uggh
> 
> ```
> 2s
> Run yarn audit --groups dependencies
> yarn audit v[1](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:1).22.19
> warning Lockfile has incorrect entry for "debug@^[4](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:5).1.0". Ignoring it.
> ┌───────────────┬──────────────────────────────────────────────────────────────┐
> │ critical      │ Babel vulnerable to arbitrary code execution when compiling  │
> │               │ specifically crafted malicious code                          │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Package       │ @babel/traverse                                              │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Patched in    │ >=7.23.2                                                     │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Dependency of │ mds                                                          │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ Path          │ mds > styled-components > @babel/traverse                    │
> ├───────────────┼──────────────────────────────────────────────────────────────┤
> │ More info     │ https://www.npmjs.com/advisories/1094349                     │
> └───────────────┴──────────────────────────────────────────────────────────────┘
> 1 vulnerabilities found - Packages audited: 306
> Severity: 1 Critical
> Done in 1.[5](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:6)7s.
> Error: Process completed with exit code 1[6](https://github.com/minio/minio/actions/runs/6537378748/job/17751086589?pr=18254#step:4:7).
> ```

Yes, I will open other PR to fix it

**Review by harshavardhana — DISMISSED** [review]

File: bundles/certified-operators/5.0.10/manifests/minio-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -571,7 +571,6 @@ spec:
                       - containerPort: 9443
                         name: https
                     resources: {}
-                    securityContext: {}
```
Comment (shtripat) [inline bundles/certified-operators/5.0.10/manifests/minio-operator.clusterserviceversion.yaml:574]: Is `securityContext` not needed ?

Comment (pjuarezd) [reply bundles/certified-operators/5.0.10/manifests/minio-operator.clusterserviceversion.yaml:574]: nope, we remove it to allow Openshift set the SecurityContext that works better for the [securityContextConstraints](https://docs.openshift.com/container-platform/3.11/architecture/additional_concepts/authorization.html#security-context-constraints) of the Openshift Project (AKA Namespace)

**Review by harshavardhana — APPROVED** [review]

**Review by jiuker — APPROVED** [review]

File: bundles/certified-operators/5.0.10/manifests/minio-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (shtripat) [reply bundles/certified-operators/5.0.10/manifests/minio-operator.clusterserviceversion.yaml:574]: ok

**Review by shtripat — APPROVED** [review]
lgtm

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**